examples/electron/package.json

```diff
@@ -45,12 +42,10 @@
     "@theia/preferences": "^0.14.0",
     "@theia/preview": "^0.14.0",
     "@theia/process": "^0.14.0",
-    "@theia/python": "^0.14.0",
     "@theia/scm": "^0.14.0",
     "@theia/search-in-workspace": "^0.14.0",
     "@theia/task": "^0.14.0",
     "@theia/terminal": "^0.14.0",
-    "@theia/textmate-grammars": "^0.14.0",
     "@theia/tslint": "^0.14.0",
     "@theia/typehierarchy": "^0.14.0",
     "@theia/typescript": "^0.14.0",
```

Comment:
we should replace typescript/tslint, that's more important to start using and testing VS Code counterparts